Docs/source/install/hpc/cori.rst

```diff
@@ -153,16 +165,19 @@ Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following comman
    cmake -S . -B build -DWarpX_OPENPMD=ON -DWarpX_DIMS=3
    cmake --build build -j 16
 
-The general :ref:`cmake compile-time options and instructions for Python (PICMI) bindings <building-cmake>` apply as usual.
+The general :ref:`cmake compile-time options and instructions for Python (PICMI) bindings <building-cmake>` apply as usual:
 
-.. warning::
+.. code-block:: bash
 
-   Consider that all three Cori partitions are *incompatible*.
+   # PICMI build
+   cd $HOME/src/warpx
 
-   Do not *source* multiple ``...warpx.profile`` files in the same terminal session.
-   Open a new terminal and log into Cori again, if you want to switch the targeted Cori partition.
+   # update Python dependencies
```

Comment:
I would even reiterate the workflow once more. Technically speaking, it's a bit risky to use `--user` - we should use a virtual environment instead.

That way, a user that switches between KNL, Haswell and the AMD partition will not run into problems. But that's not important for us right now, since you will run on KNL

Reply:
Improved :)